ui/src/components/AwsProvisionerWorkerTypeTable/index.jsx

```diff
@@ -82,26 +80,27 @@ export default class AwsProvisionerWorkerTypeTable extends Component {
       searchTerm
     );
     const iconSize = 16;
+    const headers = [
+      { label: 'Worker Type', id: 'workerType', type: 'string' },
+      { label: 'Pending Tasks', id: 'pendingTasks', type: 'number' },
+      {
+        label: 'Running Capacity',
+        id: 'runningCapacity',
+        type: 'number',
+      },
+      {
+        label: 'Pending Capacity',
+        id: 'pendingCapacity',
+        type: 'number',
+      },
+    ];
 
     return (
       <Fragment>
         <DataTable
           items={sortedWorkerTypes}
-          headers={[
-            { label: 'Worker Type', id: 'workerType', type: 'string' },
-            { label: 'Pending Tasks', id: 'pendingTasks', type: 'number' },
-            {
-              label: 'Running Capacity',
-              id: 'runningCapacity',
-              type: 'number',
-            },
-            {
-              label: 'Pending Capacity',
-              id: 'pendingCapacity',
-              type: 'number',
-            },
-          ]}
-          sortByHeader={sortBy}
+          headers={headers}
+          sortByLabel={sortBy}
```

Comment:
We'll need to do something similar to every view/component that uses the DataTable component.

Reply:
ok I 'll get to work on that. :)

Reply:
I am still working on making changes to this PR. I will have an update by tomorrow.